packages/litexa/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Litexa command-line output
+test.log
```

Comment:
This shouldn't be needed: The parent learna-level .gitignore has `.test` which contains the `test.log`, no?